lms/static/sass/_build-lms.scss

```diff
@@ -52,6 +52,7 @@
 @import 'views/decoupled-verification';
 @import 'views/shoppingcart';
 @import 'views/homepage';
+@import 'views/creative-commons';
```

Comment:
Is the CC stuff really specifically tied to a particular view? If so, placing it under the ``views/`` dir is cool, If not and used throughout several pages, it should be a component.